xs/src/libslic3r/IO/AMF.cpp

```diff
@@ -363,8 +367,9 @@ void AMFParserContext::endElement(const char *name)
         stl_allocate(&stl);
         for (size_t i = 0; i < m_volume_facets.size();) {
             stl_facet &facet = stl.facet_start[i/3];
-            for (unsigned int v = 0; v < 3; ++ v)
+            for (unsigned int v = 0; v < 3; ++ v) {
```

Comment:
The is just a whitespace change was it leftover from other changes to this loop that ended up not being necessary?

Reply:
Yes, that is the case. I'll revert.